mod/wikirate_source/spec/set/type/source/source_preview_spec.rb

```diff
@@ -49,7 +49,7 @@
         it "shows pdf" do
           file_card = @pdf_source.fetch :trait=>:file
           expect(@result).to have_tag("div", :with=>{:id=>"pdf-preview"}) do
-            with_tag "iframe", :with=>{:id=>"source-preview-iframe", :src=>"pdfjs/viewer.html?file=#{file_card.attachment.url}"}
+            with_tag "iframe", :with=>{:id=>"source-preview-iframe", :src=>"files/viewer.html?file=#{file_card.attachment.url}"}
```

Comment:
Line is too long. [128/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.